cypress/e2e/hermioneAccount.spec.cy.js

```diff
@@ -0,0 +1,65 @@
+import { faker } from '@faker-js/faker';
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  const user = 'Hermoine Granger';
+  const initialBalance = 5096;
+  const depositAmount = `${faker.number.int({ min: 500, max: 1000 })}`;
+  const withdrawAmount = `${faker.number.int({ min: 50, max: 500 })}`;
+  const balanceAfterReplenishment = initialBalance + (+depositAmount);
+  const balanceAfterWithdraw = balanceAfterReplenishment - (+withdrawAmount);
+
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('should provide the ability to work with Hermione\'s bank account', () => {
+    cy.contains('.btn', 'Customer Login').click();
+    cy.get('[name="userSelect"]').select(user);
+    cy.contains('.btn', 'Login').click();
+
+    cy.contains('[ng-hide="noAccount"]', 'Account Number')
+      .contains('strong', '1001')
+      .should('be.visible');
+    cy.contains('[ng-hide="noAccount"]', 'Balance')
+      .contains('strong', '0')

```

Comment:
The initial balance check is set to '0', which contradicts the `initialBalance` variable set to 5096. Ensure this check matches the expected initial state of the account in the application.